lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -534,6 +535,8 @@ def wrapped_function(*args, **kwargs):  # type: ignore
         name = getattr(callable, "__qualname__", getattr(callable, "__name__", str(callable)))
         with _span(f"{'(static) ' if static else ''}{qualifier} call: {name}"):  # type: ignore
             if static:
+                # fixme: do we or don't we need [1:]?
```

Comment:
yeah I'm not super clear about it either. The decorator doesn't always play nice with `@staticmethod`s. Sometimes it will receive 'self', sometimes it won't. Depending on the test run I have to add that or remove it, I'm not yet sure why.